lib/developer_portal/lib/liquid/tags/authorize_net_form.rb

```diff
@@ -1,12 +0,0 @@
-module Liquid
```

Comment:
I guess we need to leave it here, the same way we did with the ogone form liquid.